components/core/tools/packaging/universal-deb/Dockerfile

```
@@ -8,11 +8,27 @@
+# Empty default for the `ca_trust` build context, so the networked RUN below resolves its mount when
+# no host CA trust is supplied. `build.sh --with-ca-certs` overrides this stage with a staged
+# directory. Nothing is ever baked in. See yscope-dev-utils exports/docker/ca-trust.
+FROM scratch AS ca_trust
+
 FROM ${BASE_IMAGE}
 
-RUN dnf install -y epel-release \
-    && dnf install -y dpkg patchelf rpm-build \
+# dnf reads neither SSL_CERT_FILE nor CURL_CA_BUNDLE, so ca-trust-pkg-opts.sh supplies its
+# --setopt=sslcacert explicitly; the env vars container.sh exports cover curl and pip only.
+RUN --mount=type=bind,from=ca_trust,target=/run/ca-trust \
```

Hadolint (2.14.0)

[warning] Specify version with `dnf install -y <package>-<version>`.

(DL3041)